Modules/Core/Mesh/test/itkConnectedRegionsMeshFilterTest2.cxx

```diff
@@ -0,0 +1,82 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#include "itkImageFileReader.h"
+#include "itkConnectedRegionsMeshFilter.h"
+#include "itkTestingMacros.h"
+#include "itkBinaryThresholdImageFilter.h"
+#include "itkBinaryMask3DMeshSource.h"
+#include <iostream>
+
+/*
+ * Test the mesh connectivity class.
+ */
+int
+itkConnectedRegionsMeshFilterTest2(int, char * argv[])
+{
+
+  // Read 3D Image to create Test Mesh
+  using PixelType = unsigned char;
+  const unsigned int Dimension = 3;
```

Comment:
Although a minor style detail, the dimension definition is usually immediately after the input argument check block, i.e. before any other type alias definition, etc.